src/android/AndroidAccessibilityUtil.ts

```diff
@@ -1,5 +1,5 @@
 /**
-* AccessibilityUtil.ts
+* AndroidAccessibilityUtil.ts
```

Comment:
Why did you rename this file to start with "Android"? It's already in the Android directory, so adding this is redundant. It's also inconsistent with all of the other code in ReactXP. I'd prefer if you went back to the previous name for consistency.